### ISSUE TITLE
Toggle debug mode after the Viewer is created?

### ISSUE THREAD

**Issue by jp887** [openseadragon/openseadragon#1221]
Is it possible to change the debugMode after the viewer has been created? I can pass the debugMode flag via options when creating the viewer but I have a requirement to enable and disable debug mode at runtime with a keyboard shortcut.

Or, more generically, is it possible to change options after Viewer has been constructed?

**Comment by iangilman** [comment]
Changing the debugMode while running is not officially supported, but it does appear to be possible. You have to change it on each of your tiledImages and then tell the viewer to refresh itself.

```
for (var i = 0; i < viewer.world.getItemCount(); i++) {
  viewer.world.getItemAt(i).debugMode = false;
}

viewer.forceRedraw();
```

Other viewer options depend on their implementation as to whether you can change them after it's been constructed; just like debugMode, doing so is not officially supported. 

It would certainly be possible to add a `viewer.setDebugMode()` method for doing so, if you'd like to write that patch.

**Comment by jp887** [comment]
Great, thanks. I'll submit a patch!

**Comment by iangilman** [comment]
Fixed by #1224.
